### PR TITLE
Xeno tweaks and bugfixes.

### PR DESCRIPTION
**Pull request by xxalpha**
- Fixed not being able to grab things from storage while inside aliens. [https://github.com/tgstation/-tg-station/issues/9395]
 - Fixed two progress bars when devouring. [https://github.com/tgstation/-tg-station/issues/10774]
 - Reduced hunters health to 125, increased sentinels health to 150
 - Hunters are no longer transparent when stalking
 - Sentinels are transparent when stalking
 - Alien attack damage is now a constant ~~25~~ 20 brute, instead of a random value between 15 and 30 with a 10% chance to miss
 - Alien disarm stun vs cyborgs reduced to 2 seconds from 7 seconds
 - New alien caste descriptions
 - Sentinels movement speed increased to be the same as humans

edit
~~- Added ability to block alien pouncing with shields. Will weaken the alien as if it smashed against a wall.~~  THANKS PHIL SAMA

more

 - Changed facehuggers layer and alien egg layer to be MOB_LAYER to prevent players from hiding these under resin walls.

### CODE REVIEW

**Comment by RemieRichards** [comment]
>Alien attack damage is now a constant 25 brute, instead of a random value between 15 and 30
>a constant 25 brute
>90% chance for 25 brute, 10% chance for 0

HMMMMM.

**Comment by xxalpha** [comment]
That's a miss, silly.

**Comment by RemieRichards** [comment]
Which they could never do before, my point is, you've lied/hidden a change, add it to the OP.

**Comment by WJohn** [comment]
<3 <3 <3

MORE FIXES

MORE POLISH

I'll take it!

**Comment by cyfause** [comment]
I haven't seen xenos in quite a long time; when are we going to be able to see these changes in action?

**Comment by WJohn** [comment]
Xenos are around constantly in game. Admins spawn them all the time, and if not that, then miners will do that with the facehuggers they find.

File: code/modules/mob/living/silicon/silicon.dm [hunk 1]
```diff
@@ -366,7 +366,7 @@
 
 /mob/living/silicon/attack_alien(mob/living/carbon/alien/humanoid/M)
 	if(..()) //if harm or disarm intent
-		var/damage = rand(10, 20)
+		var/damage = 25
```
Comment (Fox-McCloud) [inline code/modules/mob/living/silicon/silicon.dm]: This is a huge buff.

If you're wanting to eliminate randomness, this should be ~15.

Comment (xxalpha) [inline code/modules/mob/living/silicon/silicon.dm]: Nah lol

**Comment by ike709** [comment]
"Fixes #9395"
"Fixes #10774"

**Comment by PKPenguin321** [comment]
actually it doesn't really fix either of those

**Comment by xxalpha** [comment]
It's not enough to close either of the issues.

**Comment by WJohn** [comment]
I think a constant 20 brute would be fine, frankly.

File: code/modules/mob/living/carbon/alien/larva/powers.dm [hunk 2]
```diff
@@ -40,9 +40,9 @@
...
-		L << "<span class='name'>Sentinels</span> <span class='info'>are tasked with protecting the hive and are deadly up close and at a range. They are not as physically imposing nor fast as the hunters.</span>"
-		L << "<span class='name'>Drones</span> <span class='info'>are the working class, offering the largest plasma storage and generation. They are the only caste which may evolve again, turning into the dreaded alien queen.</span>"
+		L << "<span class='name'>Hunters</span> <span class='info'>are the most agile caste tasked with hunting for hosts. They are faster than a human and can even pounce, but are not much tougher than a drone.</span>"
+		L << "<span class='name'>Sentinels</span> <span class='info'>are tasked with protecting the hive. With their ranged spit, invisibility, and high health, they make formidable guardians and acceptable secondhand hunters.</span>"
+		L << "<span class='name'>Drones</span> <span class='info'>are the weakest and slowest of the castes, but can grow into queens if there is none, and are vital to maintaining a hive with their resin secretion abilities.</span>"
```
Comment (WJohn) [inline code/modules/mob/living/carbon/alien/larva/powers.dm]: "but can grow into queens if there is none"

Should be

"but can grow into the queen if there is none"

**Pull request merged**